toolkits/mastodon/arcade_mastodon/tools/statuses.py

```diff
@@ -0,0 +1,173 @@
+from typing import Annotated
+
+import httpx
+from arcade_tdk import ToolContext, tool
+from arcade_tdk.auth import OAuth2
+from arcade_tdk.errors import RetryableToolError, ToolExecutionError
+
+from arcade_mastodon.tools.users import lookup_single_user_by_username
+from arcade_mastodon.utils import get_headers, get_url, parse_status, parse_statuses
+
+
+@tool(
+    requires_auth=OAuth2(
+        id="mastodon",
+        scopes=["write:statuses"],
+    ),
+    requires_secrets=["MASTODON_SERVER_URL"],
+)
+async def post_status(
+    context: ToolContext,
+    status: Annotated[str, "The status to post"],
```

Comment:
suggestion: might be worth mentioning that a 'status' is the text content of a user's post. Idk, worth a shot! Personally I thought status was 'online', 'offline', etc., so adding that context could be helpful.